addons/binding/org.openhab.binding.onkyo/ESH-INF/thing/thing-types.xml

```diff
@@ -85,6 +87,54 @@
 		</channels>
 	</channel-group-type>
 
+	<channel-group-type id="netMenuControls">
+		<label>Net/USB Menu</label>
+		<channels>
+            <channel id="control" typeId="netControl"/>
+			<channel id="item0" typeId="menuItem">
+                <label>Menu item 0</label>
```

Comment:
could you please run the auto-formatter, so that the spaces are replaced by tabs?